htdocs/theme/eldy/style.css.php

```diff
@@ -1357,6 +1357,23 @@
 	}
 }
 
+/* Force values on one colum for small screen */
```

Comment:
The 'fichehalfleft' and 'fichehalfright' are already 2 class to automatically switch on 1 column where scrren is too small.
![image](https://user-images.githubusercontent.com/883887/48220487-b289bd80-e38f-11e8-83df-944f8554130d.png)

The div <div class="div-table-responsive-no-min"> was missing. I added it.
Is it enough now on your side.